_co-op-companies/L-247.md

```diff
@@ -5,10 +5,10 @@ description: "24-7 Intouch provides contact center solutions."
 link: "http://24-7intouch.com/"
 image: "assets/img/logos/247-in-touch.png"
 width: "100px"
-complete: false
+complete: true
 members:
-  - name: "UMWics Member"
-    summary: "UMWICS Member worked at 24-7 Intouch."
-    statement: "UMWics Member had a good time."
-    image: "/assets/img/co-op/person.jpg"
+  - name: "Katharine King"
+    summary: "Kat did her third work term at 24-7 Intouch."
+    statement: "She was on the Robotics Processing Automation Team where she designed client's interfaces using C# and XAML. She also worked as a backend developer using Selenium and Javascript to communicate with the client's web server."

```

Comment:
There's a typo somewhere, either the sentence should be "she designed **the** client's interfaces" or "designed clients' interfaces" depending on if there were multiple clients (the end of the sentence seems to indicate just one).
Javascript -> JavaScript